README.md

```diff
@@ -122,14 +122,22 @@ You can also add `Ignite UI for Angular` components to your projects by using th
 
 See the [Contribution guide](https://github.com/IgniteUI/igniteui-cli/blob/master/.github/CONTRIBUTING.md) to get started.
 
+### Prerequisites
+The repository houses multiple packages and orchestrates building and publishing them with [lerna](https://github.com/lerna/lerna) and [yarn](https://yarnpkg.com/).
```

Comment:
```suggestion
The repository houses multiple packages and orchestrates building and publishing them with [lerna](https://github.com/lerna/lerna) and [yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/).
```
Perhaps this description along with the "Packages" section can be moved on top, just before table of contents (pending further cleanup)